src/main/java/io/anserini/index/IndexUtils.java

```diff
@@ -427,40 +425,6 @@ public void run() {
     LOG.info(String.format("Raw documents are output to: %s", outputDir));
   }
 
-  public String getTransformedDocument(String docid) throws IOException, NotStoredException {
```

Comment:
Why are we removing these again? Is it because you got rid of the LuceneDocId stuff in the last PR?

Reply:
The transformer is the thing that converts "raw" to "contents", it's not relevant anymore based on our refactoring plans, so getting rid of it while I'm at it.

In general `IndexUtils` is legacy, we've been adding new functionality to `IndexReaderUtils`.